gridstatus/pjm.py

```diff
@@ -1158,7 +1171,46 @@ def get_solar_forecast(self, date, end=None, verbose=False):
 
         return self._parse_solar_forecast(df)
 
-    def _parse_solar_forecast(self, df):
+    @support_date_range(frequency=None)
+    def get_solar_forecast_5min(
```

Comment:
in the rest of the code base we typically add a space to `5_min`, so it is `get_solar_forecast_5_min`

Reply:
done